javascripts/govuk/details.polyfill.js

```diff
@@ -234,6 +234,15 @@
     init: function ($container) {
       GOVUK.details.addEvent(document, 'DOMContentLoaded', GOVUK.details.addDetailsPolyfill)
       GOVUK.details.addEvent(window, 'load', GOVUK.details.addDetailsPolyfill)
+
+      if (window.GOVUK.analytics && window.GOVUK.analytics.trackEvent) {
+        if (window.location.origin === 'https://gov.uk') {
```

Comment:
Just to check, because I don't know, should this be `https://www.gov.uk`?

Also wondering if this is only a temporary branch and will never be merged why we need this check?

Reply:
This has been removed.